hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -70,17 +75,22 @@ public class OmMetadataGenerator extends BaseFreonGenerator
 
   enum Operation {
     CREATE_FILE,
+    CREATE_STREAM_FILE,
     LOOKUP_FILE,
     READ_FILE,
     LIST_STATUS,
     CREATE_KEY,
+    CREATE_STREAM_KEY,
     LOOKUP_KEY,
     GET_KEYINFO,
     HEAD_KEY,
     READ_KEY,
     LIST_KEYS,
+    LIST_KEYS_LIGHT,
     INFO_BUCKET,
     INFO_VOLUME,
+    EMPTY_RPC_WRITE_RATIS,
+    EMPTY_RPC,
```

Comment:
Do we need those? They're already available in `OmRPCLoadGenerator`. They don't seem to fit the category of "MetadataGenerator".

Reply:
Looks like the name is unfortunate but the tool as it exists is supposed generate metadata load vs metadata content. I am ok with adding this here. Freon does have overall in its functionality for load generation.